TracNghiem/database/migrations/2022_04_01_014903_create_user_exams_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_exams', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade')->onUpdate('cascade');
+            $table->unsignedBigInteger('exam_id');
+            $table->foreign('exam_id')->references('id')->on('exams')->onDelete('cascade')->onUpdate('cascade');
+            $table->float('percentage', 8, 2)->default(0);
+            $table->boolean('is_passed')->default(false);
+            $table->timestamps();
```

Comment:
với cấu trúc bảng như thế này thì có thể xem lại toàn bộ bài kiểm tra của user k em. Tức là xem câu 1 ý gì, đúng hay sai